domains/ram.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "personal use",
+    "repo": "https://github.com/thisisramhere/Portfolio",
+    "owner": {
+        "username": "thisisramhere",
+        "email": "ramkiranmohan1759@gmail.com",
+        "twitter": "RamKiranMohan1"
+    },
+    "record": {
+        "CNAME": "thisisram.github.io"
```

You added the wrong CNAME for github pages (its usually your username and then github.io)
```suggestion
        "CNAME": "thisisramhere.github.io"
```